src/renderer/components/menu/menu.tsx

```diff
@@ -260,6 +260,8 @@ export class Menu extends React.Component<MenuProps, State> {
   }
 
   onBlur() {
+    if (!this.isOpen) return;
```

Comment:
Performance analysis shows that menu instances triggers a lot of `onBlur()` events when unmounting.

Reply:
Maybe add a comment to that effect then.